changing ADDR_Address_TEST name

m-kinkade opened the pull request:
# Critical Changes

# Changes
Renamed ADDR_Addresses_TEST class to ADDR_Address_TEST.
ADDR_Addresses_TEST was used previously and subsequently deprecated and replaced with ADDR_Account_TEST.
Renaming to ADDR_Address_TEST to prevent build error related to deprecated class name.

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

spinkelman commented:
No Errors in the Build in MetaCI with Multiple types of flows. Passing QA

xinnanlisf commented:
@juliesnow this is an internal PR we create to fix a build failure. We are not sure if we need have anything in the changes section but thinking to let you decide. :)

m-kinkade commented:
Because this is an internal item, I've removed the following changes from release notes:

Renamed ADDR_Addresses_TEST class to ADDR_Address_TEST.
ADDR_Addresses_TEST was used previously and subsequently deprecated and replaced with ADDR_Account_TEST.
Renaming to ADDR_Address_TEST to prevent build error related to deprecated class name.
